packages/metadataform-extension/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "@jupyterlab/metadataform-extension",
+  "version": "4.0.0-alpha.14",
```

Probably this should be bumped to be aligned with the latest release:

```suggestion
  "version": "4.0.0-alpha.15",
```